projects/ag-grid/documentation/code-examples/data-entry-grid/basic/data-entry-grid-docs-demo.component.ts

```diff
@@ -0,0 +1,162 @@
+import {
+  ChangeDetectionStrategy,
+  ChangeDetectorRef,
+  Component,
+} from '@angular/core';
+import { SkyCellType, SkyAgGridService } from '@skyux/ag-grid';
+import { SkyModalService, SkyModalCloseArgs } from '@skyux/modals';
+
+import {
+  GridApi,
+  GridReadyEvent,
+  GridOptions,
+  ValueFormatterParams,
+} from 'ag-grid-community';
+
+import { SkyDataEntryGridContextMenuComponent } from './data-entry-grid-docs-demo-context-menu.component';
+import { SKY_AG_GRID_DEMO_DATA } from './data-entry-grid-docs-demo-data';
+import { SkyDataEntryGridEditModalContext } from './data-entry-grid-docs-demo-edit-modal-context';
+import { SkyDataEntryGridEditModalComponent } from './data-entry-grid-docs-demo-edit-modal.component';
+
+@Component({
+  selector: 'app-data-entry-grid-docs-demo',
+  templateUrl: './data-entry-grid-docs-demo.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class SkyDataEntryGridDemoComponent {
+  public gridData = SKY_AG_GRID_DEMO_DATA;
+  public columnDefs = [
+    {
+      field: 'selected',
+      type: SkyCellType.RowSelector,
+    },
+    {
+      colId: 'context',
+      headerName: '',
+      maxWidth: 50,
+      sortable: false,
+      cellRendererFramework: SkyDataEntryGridContextMenuComponent,
+    },
+    {
+      field: 'name',
+      headerName: 'Name',
+    },
+    {
+      field: 'age',
+      headerName: 'Age',
+      type: SkyCellType.Number,
+      maxWidth: 60,
+    },
+    {
+      field: 'startDate',
+      headerName: 'Start Date',
```

Comment:
Same note as above about sentence-style capitalization in the UI.
```suggestion
      headerName: 'Start date',
```